doc/man7/openssl-core.h.pod

```diff
@@ -30,7 +30,7 @@ Arrays of this type must be terminated with a tuple having function
 identity zero and function pointer NULL.
 
 The available function identities and corresponding function
-signatures are defined by L<openssl-core_numbers.h(7)>.
+signatures are defined in C<openssl/core_numbers.h>.
```

Comment:
Actually, no.  But this is a good reminder that I need to start on `doc/man7/openssl-core-numbers.h`, thank you.

Reply:
Ping

Reply:
fixed, pushed.